.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": false,
```

Comment:
I personally find better to use single quotes in javascript and double quotes in html/jsx elements. Apart from that, looks fine :)